CSL-Traffic/Initializer.cs

```diff
@@ -1810,7 +1687,7 @@ public static void DumpTextureToPNG(Texture previewTexture, string filename = nu
                 }
                 catch (UnityException)
                 {
```

Comment:
By searching through the diff for catch you can see the clauses I mention. Here is one.